ci/codespell-ignore-words.txt

```diff
@@ -4,13 +4,18 @@ ba
 cannotation
 coo
 curvelinear
+dedented
+falsy
 flate
+hax
 hist
+inh
```

Comment:
Use in 

https://github.com/matplotlib/matplotlib/blob/23730b41febeb3cc80bd4733a228f0ed22e1b6d5/lib/matplotlib/transforms.py#L2562

for "invalid height" is reasonable, so added.